### PR TITLE
Update reporting.py

### PR DESCRIPTION
**Pull request by CustomIcon**
Patch for Admins and Sudoers to be immune to Reporting

### CODE REVIEW

File: haruka/modules/reporting.py [hunk 1]
```diff
@@ -10,8 +10,10 @@
 from haruka.modules.log_channel import loggable
 from haruka.modules.sql import reporting_sql as sql
 from haruka.modules.tr_engine.strings import tld
+from haruka.modules.helper_funcs.chat_status import bot_admin, user_admin
```
Comment (RealAkito) [inline haruka/modules/reporting.py:13]: Codacy Issue found: [redefinition of unused 'user_admin' from line 9 (F811)](https://app.codacy.com/gh/HarukaNetwork/HarukaAya/pullRequest?prid=5652154)

**Comment by RealAkito** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- haruka/modules/reporting.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/HarukaNetwork/HarukaAya/pullRequest?prid=5652154&bid=18506490)

File: haruka/modules/reporting.py [hunk 2]
```diff
@@ -61,6 +63,10 @@ def report(bot: Bot, update: Update) -> str:
         reported_user = message.reply_to_message.from_user  # type: Optional[User]
         chat_name = chat.title or chat.first or chat.username
         admin_list = chat.get_administrators()
+
```
Comment (RealAkito) [inline haruka/modules/reporting.py:66]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/gh/HarukaNetwork/HarukaAya/pullRequest?prid=5652154)

File: haruka/modules/reporting.py (hunk 1 above)
Comment (RealAkito) [inline haruka/modules/reporting.py:13]: Codacy Issue found: [Reimport 'user_admin' (imported line 9)](https://app.codacy.com/gh/HarukaNetwork/HarukaAya/pullRequest?prid=5652154)

Comment (CustomIcon) [reply haruka/modules/reporting.py:13]: oof... i forgot to remove the user_admin from line 9... if pushing to haruka aya master branch please remove it kthx

**Comment by RealAkito** [comment]
Closing, behavior intended, we moved to Gitlab.

**Pull request closed without merging**